CHANGES.txt

```diff
@@ -15,6 +15,9 @@ the appropriate section of the docs.
 Breaking Changes
 ================
 
+ - Changed default required shards for index creation and write operations
+   from quorum to ``all``.
```

Comment:
pls note that we introduced a new table setting `write.wait_for_active_shards` to give the users the possibility to adjust this on their needs. also pls write "required shard copies". pls use `table` not index